litmus/director/openebs-reinstallation/README.md

```diff
@@ -0,0 +1,181 @@
+# openebs-reinstallation-check
+
+<b>tcid:</b> iuoi06<br>
+<b>name:</b> Install OpenEBS on a machine already OpenEBS is installed with same version using operator.yaml<br>
+<b>sidebar_label:</b> openebs-reinstallation-check
+
+
+## Experiment Metadata
+
+<table>
+  <tr>
+    <th> Type </th>
+    <th> Description </th>
+    <th> Tested K8s Platform </th>
+  </tr>
+  <tr>
+    <td> Openebs Reinstallation </td>
+    <td> openebs-reinstallation-check</td>
+    <td> GKE </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Along with k8s, Litmus should be installed in the cluster.
+- Every component of the cluster should be healthy and running.
+- Ensure that the `openebs-reinstallation-check` resource is available in the cluster.
+
+## Entry Criteria
+
+- Kubernetes nodes are up and running with no openebs installed in the cluster .
+
+## Exit Criteria
+
+- Kubernetes nodes are up and running with openebs installed twice in the cluster . 
```

Comment:
Openebs will not be installed the next time. So technically the line is not correct.